content/finding-command-help.md

```diff
@@ -0,0 +1,109 @@
+
+### --help
+
+`--help` prints out a shortened version of the man page, usually including a brief example and all options for a command. `--help` prints output to the terminal session without navigating away from the shell prompt.
+
+**Use case**: A user wants to quickly reference all options for a known command in the current Terminal session.
+
+**Usage**: Type a command + `--help`, then hit <kbd>Enter</kbd>. This example uses the `cd` command. Commands must be an exact match.
+```bash
+cd --help
+```
+![--help output](/images/finding-command-help/--help-output.png)
+
+### whatis
+
+`whatis` provides a very short description of a command's functionality.
+
+**Use case**: A user wants a very brief reminder of a known command's functionality printed in the current Terminal session.
+
+**Usage**: Type `whatis` + command, then hit <kbd>Enter</kbd>. This example uses the `mkdir` command. Commands must be an exact match.
+```bash
+whatis mkdir
+```
+![whatis output](/images/finding-command-help/what-output.png)
+
+### which
+
+`which` provides the directory location of a command.
+
+**Use case**: A user would like to know which installed package provides a given command.
+
+**Usage**: Type `which` + command(s), then hit <kbd>Enter</kbd>. This example uses the ls and dpkg commands.
+```bash
+which ls -l dpkg -S
+```
+![which output](/images/finding-command-help/which-output.png)
+
+### apropos
+
+`apropos` performs a keyword search on command man pages. This is useful when searching for a command by its function. `apropos` prints output to the terminal session without navigating away from the shell prompt.
+
+**Use case**: A user can't remember the specific name of a command, but knows keywords commonly used to describe the command's functions.
+
+**Usage**: Type `apropos` + keyword, then hit <kbd>Enter</kbd>. This example searches for commands related to the `reboot` keyword.
+```bash
+apropos reboot
+```
+![apropos output](/images/finding-command-help/apropos-output.png)
+
+### man
+
+`man` is short for manual pages. A command's manual page is a consistently structured text file describing the command's function, usage syntax, options, and similar commands. `man` opens a manpage in the terminal session, and pressing <kbd>q</kbd> exits the manpage and returns the user to the shell prompt.
+
+**Use case**: A user wants to see actions performed by a specific command, and available options to modify the outcome of the command.
+
+**Usage**: Type `man` + any command into a Terminal session, then hit <kbd>Enter</kbd>. This example uses the `reboot` command.
+```bash
```

Comment:
There need to be line-breaks between **Usage** line, codeblock and png, otherwise it may not pass linter.

Reply:
It has already passed the linter; we had to [disable that lint](https://github.com/system76/docs/pull/915#discussion_r802975777) because it prevented having code blocks within ordered or unordered list items.

However, it would still be good practice to have the line break in cases like this where it's not part of a list item.

Reply:
Thanks for clarifying